pages/locaties-selecteren.md

```diff
@@ -1,7 +1,7 @@
 ---
-title: Locaties
-description: Locaties selecteren en reserveren
-permalink: /locaties/
+title: Locaties selecteren
+description: Locaties bekijken en selecteren
+permalink: /locaties-selecteren/
```

Comment:
Deze pagina zou ik op `locaties` houden (korter en eenvoudiger). De naam in de navigatie kan je aanpassen.

Reply:
Oke, ik was niet zeker of dat hetzelfde moest zijn van naam. Ik pas het aan.

Reply:
Nee, met de titel kan je wat spelen (en kan eventueel zelf verschillend zijn van de titel in de navigatie), maar de `permalink` wijzigen we best niet na lancering (en houden we ook best kort), omdat mensen die kunnen bookmarken of naar verwijzen.